yodaconvert/src/components/Converter.js

```diff
@@ -1,24 +1,39 @@
 import React, {Component} from "react";
 import '../App.css';
+import convertToHTML from 'markdown-to-html-converter' //npm install markdown-to-html-converter
 import ButtonConvDel from './Button.jsx';
 
 
 class Components extends Component {
     constructor (props) {
         super(props)
-        this.state={}
+        this.state={
+            inputText:'',
+            outputText:''
+        }
+        this.handleInputChange = this.handleInputChange.bind(this)
     }
+
+
+handleInputChange(e) {
+    this.setState({ 
+        inputText: e.target.value,
+        outputText : convertToHTML(this.state.inputText) })
+      }
+    
 render () {
+    console.log(this.state)
     return (
         <div className="textContainer">
-            <textarea className="input-text" rows="6" cols="40"></textarea>
+            
+            <textarea className="input-text" name="inputText" rows="6" cols="40" value={this.state.inputText} onChange={this.handleInputChange}>    
+            </textarea>
             <ButtonConvDel/>
-            <textarea className="output-text" rows="6" cols="40"></textarea>
+            <textarea className="output-text" name="outputText" rows="6" cols="40" value={this.state.outputText} onChange={this.handleInputChange} >    
```

Comment:
shouldn't this one be read only?

see https://www.w3schools.com/tags/att_textarea_readonly.asp